### DISCUSSION TITLE
[app directory] Routing doesn't work inside not-found.tsx component

### DISCUSSION THREAD

**Discussion by vixero-dev** [vercel/next.js#49054] [Help] [answered]
### Summary

None of the routing options are working inside the built-in [not-found](https://beta.nextjs.org/docs/api-reference/file-conventions/not-found) component.

### Additional information

Navigate to `/asd` to get redirected to the default `not-found` component. Click on `Back Home` button. Nothing happens.

I can make the navigation work by using `window.location.href` instead, but this doesn't feel right.

### Example

https://stackblitz.com/edit/vercel-next-js-wjgc8g?file=app%2Fnot-found.tsx

**Comment by tobidemski** [comment]
Did you find a fix for that without using window.location?

**Reply by vixero-dev** [reply]
No, I haven't. At this point this is a known issue and looking at the PR's, #49855 might have fixed it but I haven't had the time to test it.

**Comment by icyJoseph** [comment, marked as answer]
It works now. Tested with next@13.4.3 -
